resources/views/dashboard/controllers/calendar.blade.php

```diff
@@ -5,14 +5,12 @@
 @endsection
 
 @section('content')
-<div class="container-fluid view-header">
-    @if($calendar->type == 1)
-        <h4>Viewing Calendar Event, "{{ $calendar->title }}"</h4>
-    @else
-        <h4>Viewing News Posting, "{{ $calendar->title }}"</h4>
-    @endif
-</div>
-<br>
+@if($calendar->type == 1)
+    @include('inc.header', ['title' => 'Viewing Calendar Event, "' . $calendar->title . '"'])
+@else
+    @include('inc.header', ['title' => 'Viewing News Posting, "' . $calendar->title . '"'])
```

Comment:
I'd like to be maintain consistency with how we are inserting variables here. Some places were concatenating and other places using the double curly braces. Is there any reason we have to use one or the other in some places and not in others?

Reply:
@iccowan Yes, the double curly braces are used in a few limited cases where we're inserting HTML. I've used cat everywhere else for simplicity.

Reply:
Got it okay that makes sense. I'll QA now